pkg/fleetallocation/controller.go

```diff
@@ -288,3 +292,26 @@ func (c *Controller) allocate(f *stablev1alpha1.Fleet) (*stablev1alpha1.GameServ
 
 	return gs, nil
 }
+
+// patch the labels and annotations of an allocated GameServer with metadata from a FleetAllocation
+func (c *Controller) patchMetadata(gs *stablev1alpha1.GameServer, fam *stablev1alpha1.FleetAllocationMeta) () {
+	// patch ObjectMeta labels
+	if fam.Labels != nil {
+		if gs.ObjectMeta.Labels == nil {
+			gs.ObjectMeta.Labels = make(map[string]string, 1)
```

Comment:
@enocom should this be `make` or should this be `gs.ObjectMeta.Labels = map[string]string{}` ?

Reply:
There almost synonymous. I say go with whatever you do elsewhere in the codebase. The use of `make` looks a little less common to my eyes.